plugins/com.google.cloud.tools.eclipse.appengine.newproject/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.ui.ide,
  org.eclipse.jst.j2ee.web
 Import-Package: com.google.cloud.tools.eclipse.appengine.localserver.runtime,
+ com.google.cloud.tools.eclipse.appengine.ui,
```

Comment:
You don't actually technically need to reference a bundle to pull in these references within the plugin.xml.  But such a reference does ensure that the other bundle will be pulled in as a dependency. But since we're referencing resources within the bundle, rather than classes from a package, then I think this might be better as a Require-Bundle.